src/renderer/components/layout/sidebar.tsx

```diff
@@ -46,7 +46,7 @@ export class Sidebar extends React.Component<Props> {
 
   renderCustomResources() {
     if (crdStore.isLoading) {
-      return <Spinner centerHorizontal/>;
+      return <Spinner centerHorizontal />;
     }
```

Comment:
The desired effect could be done with
```
if (crdStore.isLoaded) {
      return (
        <div className="flex justify-center">
          <Spinner/>
        </div>
      );
    }
```

Then we can completely remove any references (and styles) for `centerHorizontal`.
![sidebar spinner](https://user-images.githubusercontent.com/9607060/112962862-808b7880-914f-11eb-9d11-809e6f0099e6.gif)

Reply:
Do you mean that we should remove the `centerHorizontal` prop from `<Spinner>`?

Reply:
Yes, we can do this since no one using it.